src/default.js

```diff
@@ -29,6 +29,20 @@ import VueRouter   from 'vue-router';
 Vue.use(VueEvents);
 Vue.use(VueRouter);
 
+// --- Gettext ----
+
+import GetTextPlugin from 'vue-gettext';
+import translations from '../l10n/translations.json'
+
+Vue.use(GetTextPlugin, {
+  availableLanguages: {
```

Comment:
can we extract available languages from the translations.json?
otherwise we have to touch always two locations when adding a language.

Reply:
This information is hardcoded in the l10n/Makfile currently.
We could change this behaviour as soon as we use webpack as building system, because then this information is inside the javascript scope.

At this point it's a little tricky, because the vue-gettext plugin want's this information to map available locales to language names& to json content string.
We could simply parse all available languages from the json file on requiring it, but this wouldn't be a fast & clean solution.

Reply:
ok - let's keep this for a future PR. THX